backend/app/cmd/server.go

```diff
@@ -493,8 +483,12 @@ func (s *ServerCommand) newServerApp(ctx context.Context) (*serverApp, error) {
 			telegramBotUsername = telegramService.GetBotUsername()
 		}
 		notifyDestinations = append(notifyDestinations, telegramService)
+		telegramReceivers := []providers.TGUpdatesReceiver{telegramService}
```

Comment:
just a suggestion/idea -  maybe it make sense to extract all/some tg related code to a separate func prepping all the tg related things? i.e both tg auth, tg notify as well as the dispatcher. As of now tg prep all around in this newServerApp and I would like to keep this func as close as possible to a glue code only (call smth, check err and  assign to smth)